Packages/MIES/MIES_Utilities.ipf

```diff
@@ -1995,6 +1995,25 @@ Function/S ListFromList(list, itemBegin, itemEnd, [listSep])
 	return list[start, stop - 1]
 End
 
+/// @brief Return the minimum and maximum of both values
+Function [variable minimum, variable maximum] MinMax(variable a, variable b)
+
+	minimum = min(a, b)
+	maximum = max(a, b)
+End
+
+/// @brief Return a new wave from the subrange of the given 1D wave
+Function/WAVE DuplicateSubRange(wv, first, last)
+	WAVE wv
+	variable first, last
+
+	ASSERT(DimSize(wv, COLS) == 0, "Requires 1D wave")
+
+	Duplicate/RMD=[first, last]/FREE wv, result
+
+	return result
+End
+
```

Comment:
I need to remember we have these as i've written the code contained within sooo many times.

Reply:
Yes keeping all of them remembered is a challenge of its own.